Sources/SuperValidator/Validators/Phone.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  File.swift
+//  
+//
+//  Created by christopher.mienarto on 23/03/22.
+//
+
+import Foundation
+
+public enum PhoneFormat {
+    case international
+    case nanp
+    case epp
```

Comment:
Maybe we can add comment as well here? What nanp and epp stand for is

Reply:
got it mas